decentraland/unity-explorer
# Feat: color picker

## What does this PR change?

This PR adds the color picker in the backpack, allowing to change hair/eyes/skin colors
...

## How to test the changes?


For QA: the UI/UX aspect can be a bit unpolished but it's not a blocking problem, it will be refined in another PR, check only what follows: 
1. Launch the explorer
2. Go to backpack
3. Select hair/eyes/bodyshape section
4. Verify that changing color in the color picker affects the character preview
5. Verify that it deploys the changes correctly when closing the backpack

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
